docs/how_to/use_as_tsdb.md

```diff
@@ -0,0 +1,155 @@
+# Using M3DB as a generic time series database
+
+## Overview
+
+M3 has native integrations that make it particularly easy to use it as a metrics storage for [Prometheus](../integrations/prometheus.md) and [Graphite](../integrations/graphite.md). However, M3DB can also be used as a stand alone time series database.
+
+## Data Model
+
+### IDs and Tags
+
+The data model provided by M3DB is that each cluster can contain multiple namespaces. Each [namespace can be configured and tuned independently.](../operational_guide/namespace_configuration.md)
+
+Each namespace can also be configure with its own schema (see `"Schema Modeling" section below).
+
+Within a namespace, each time series is uniquely identified by an ID which can be any valid string / byte array. In addition, tags can be attached to any time series which makes the time series queryable using the inverted index.
+
+M3DB's inverted index supports term (exact match) and regular expression queries over all tag values, and individual tag queries can be arbitrarily combined using `AND`, `OR`, and `NOT` operators.
+
+For example, imagine you were developing an application to track a fleet of vehicles. One way to structure your time series could be as follows:
+
+| Time Series ID | "type" tag value | "city" tag value          |
+|----------------|------------------|---------------------------|
+| vehicle_id_1   | sedan            | san_francisco             |
+| vehicle_id_2   | bicycle          | san_francisco             |
+| vehicle_id_3   | scooter          | chicago_region_0          |
+| vehicle_id_4   | scooter          | chicago_region_1          |
```

Comment:
Agree that  probably reads a little cleaner.

Also do you maybe want to just add a third tag for region/hexagon ID or something instead of using `region_0` and `region_` to differentiate the two chicago areas?

Reply:
Yeah sure I was just quickly trying to find an excuse to need a regex query, your suggestion makes more sense. I’ll invert the table as Simon demonstrated too